packages/app/src/app/components/CreateNewSandbox/CreateSandbox/FromTemplate.tsx

```diff
@@ -0,0 +1,98 @@
+import { useAppState } from 'app/overmind';
+import React, { useState } from 'react';
+import {
+  Stack,
+  Element,
+  Checkbox,
+  Icon,
+  Button,
+} from '@codesandbox/components';
+
+import { SelectContainer, StyledInput, StyledLabel } from './elements';
+
+export const FromTemplate = ({ onCancel }) => {
+  const { hasLogIn, user, dashboard, activeTeam } = useAppState();
+
+  // TODO: Set generated name as default value if we can / need
+  // otherwise tell the user if empty we generate a name
```

Comment:
Doing this by describing the Sandbox name input with "Leaving the field empty will generate a random name".